abel/tools/vmi.py

```diff
@@ -243,3 +243,70 @@ def PAD(theta, beta, amplitude):
         amplitude, error_amplitude = np.nan, np.nan
 
     return (beta, error_beta), (amplitude, error_amplitude)
+
+
+def toPES(radial, intensity, wavelength, Vrep, R2E, zoom=1, energy_form='eBE',
+          units='cm'):
+    """
+    Convert speed radial coordinate into electron binding/kinetic energy.
+    Return the photoelectron spectrum (PES).
+
+    NB requires experiment parameters, measurement `wavelength`,
```

Comment:
285-287 has:
>             # This "previous" slice corresponds to the previous slice intensity
            # profile that has # been re-scaled. Thus, if the next slice is 
            # identical, it will be assigned a scale factor of 1.0